src/main/scala/uncore/tilelink2/Isolation.scala

```diff
@@ -55,9 +56,10 @@ class TLIsolation(fOut: (Bool, UInt) => UInt, fIn: (Bool, UInt) => UInt) extends
 object TLIsolation
 {
   // applied to the TL source node; y.node := TLIsolation(fOut, fIn)(x.node)
-  // f should insert an isolation gate between the input UInt and its result
-  // fOut is applied for data flowing from client to manager
-  // fIn  is applied for data flowing from manager to client
+  // f* should insert an isolation gate between the input UInt and its result
+  // fOut is applied to data flowing from client to manager
+  // fIn  is applied to data flowing from manager to client
+  // >>>> WARNING: the isolation functions must bring the values to 0 <<<<
```

Comment:
The isolation needs to use '0' so that you know the counters are 0 and that the reset_n is 0.